lib/iiif/v3/presentation/canvas.rb

```diff
@@ -3,25 +3,22 @@ module V3
     module Presentation
       class Canvas < IIIF::V3::AbstractResource
 
-        # TODO (?) a simple 'Image Canvas' constructor.
-
-        TYPE = 'Canvas'
+        TYPE = 'Canvas'.freeze
 
         def required_keys
           super + %w{ id label }
         end
 
-        def array_only_keys
-          super + %w{ content }
+        def prohibited_keys
```

Comment:
great question!  So the `int_only_keys` that were here in Canvas are now in `abstract_resource`;  `prohibited_keys` is a new thing (partly because so many keys are in `abstract_resource` and partly because the v3.0 spec is full of "this key MUST not be used elsewhere" language);  legal_viewing_hint_values are potentially unique to different classes so they need to be specified at this level, and I used the values indicated by the current v3.0 draft.